Week1/homework/js-exercises/basicMath.js

```diff
@@ -0,0 +1,9 @@
+If x equals 7, and the only other statement is x = x % 3, what would be the value of x after the calculation?
+//divide 7 by 3, remainder is 1 hence is the value of x. % is the modulus operator, this statement gives x the value of remainder.
+
+If y equals 21, and the only other statement is y = y % 4, what would be the value of y after the calculation?
+//divide 21 by 4, remainder is 1 hence is the value of y. % is the modulus operator, this statement gives y the value of remainder.
+
+
+If z equals 13, and the only other statement is z = z % 2, what would be the value of z after the calculation?
+//divide 13 by 2, remainder is 1 hence is the value of z. % is the modulus operator, this statement gives z the value of remainder.
```

Comment:
👍 for explanation but there is a small confusion here which I am checking with Noer currently.